### PR TITLE
feat/rspack react tractor 2

### PR DESCRIPTION
**Pull request by ryok90**

### CODE REVIEW

File: examples/react-rspack-tractor-2.0/apps/checkout/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  },
+  "devDependencies": {
+    "@module-federation/enhanced": "0.2.6",
+    "@rspack/cli": "^0.7.5",
+    "@rspack/core": "^0.7.5",
+    "@types/canvas-confetti": "1.6.4",
+    "@types/react": "^18",
+    "@types/react-router-dom": "^5",
+    "typescript": "^5",
+    "zephyr-webpack-plugin": "../../../../../zephyr-mono/libs/zephyr-webpack-plugin/dist"
```
Comment (zmzlois) [inline examples/react-rspack-tractor-2.0/apps/checkout/package.json:36]: remember to update it to our installable plugin but not the one on our machine xo

**Review by zmzlois — CHANGES_REQUESTED** [review]

File: examples/react-rspack-tractor-2.0/apps/app/package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    "@module-federation/enhanced": "0.2.6",
+    "@rspack/cli": "^0.7.5",
+    "@rspack/core": "^0.7.5",
+    "@rspack/plugin-react-refresh": "^0.7.5",
+    "@types/node": "^20",
+    "@types/react": "^18.3.3",
+    "@types/react-dom": "^18",
+    "react-refresh": "^0.14.2",
+    "typescript": "^5",
+    "zephyr-webpack-plugin": "../../../../../zephyr-mono/libs/zephyr-webpack-plugin/dist"
```
Comment (zmzlois) [inline examples/react-rspack-tractor-2.0/apps/app/package.json:33]: remember to update it to our installable plugin and try build it from there xx

File: examples/react-rspack-tractor-2.0/apps/decide/package.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    "react-router-dom": "^6.26.0"
+  },
+  "devDependencies": {
+    "@module-federation/enhanced": "0.2.6",
+    "@rspack/cli": "^0.7.5",
+    "@rspack/core": "^0.7.5",
+    "@types/react": "^18",
+    "@types/react-router-dom": "^5",
+    "typescript": "^5",
+    "zephyr-webpack-plugin": "../../../../../zephyr-mono/libs/zephyr-webpack-plugin/dist"
```
Comment (zmzlois) [inline examples/react-rspack-tractor-2.0/apps/decide/package.json:32]: same as this

File: examples/react-rspack-tractor-2.0/apps/explore/package.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    "react-router-dom": "^6.26.0"
+  },
+  "devDependencies": {
+    "@module-federation/enhanced": "0.2.6",
+    "@rspack/cli": "^0.7.5",
+    "@rspack/core": "^0.7.5",
+    "@types/react": "^18",
+    "@types/react-router-dom": "^5",
+    "typescript": "^5",
+    "zephyr-webpack-plugin": "../../../../../zephyr-mono/libs/zephyr-webpack-plugin/dist"
```
Comment (zmzlois) [inline examples/react-rspack-tractor-2.0/apps/explore/package.json:32]: same as this

File: examples/react-rspack-tractor-2.0/apps/checkout/package.json (hunk 1 above)
Comment (ryok90) [reply examples/react-rspack-tractor-2.0/apps/checkout/package.json]: stop reviewing draft pr's you crazyyy

**Review by zmzlois — APPROVED** [review]

**Pull request merged**